src/ui/ui_nav_link.js

```diff
@@ -12,6 +13,6 @@ export default class UiNavLink {
   }
 
   toJSON() {
-    return pick(this, ['title', 'url', 'order', 'description', 'icon', 'linkToLastSubUrl']);
+    return pick(this, ['id', 'title', 'url', 'order', 'description', 'icon', 'linkToLastSubUrl']);
```

Comment:
Can you add a tests for toJSON?  People may rely on these properties existing, so it'd be good to know that one doesn't get accidentally removed or something.